### PR TITLE
[JENKINS-70169] Add missing breadcrumb items in `resources/jenkins`

### PR DESCRIPTION
**Pull request by lemeurherve**
Extract from #7457 

Follow-up of https://github.com/jenkinsci/jenkins/pull/6912

Initially fixing [JENKINS-70169](https://issues.jenkins.io/browse/JENKINS-70169), I've included every missing breadcrumb items I've found by searching for "sidepanel.jelly" in jelly files.

Notes: 
- ~~I've added `<!-- no need for breadcrumb here -->` as comment in views where adding a breadcrumb item wasn't necessary, let me know if it's preferable to omit them.~~
As noticed in the review comments, the presence of `f:breadcrumb-config-outline` tag is enough to indicate page not needing an additional breadcrumb.

- This doesn't fix missing breadcrumb depending on plugin (ex: https://github.com/jenkinsci/cloudbees-folder-plugin/pull/278)

### Testing done

Built and ran Jenkins locally, going on modified pages to check the change.

There is no additional automatic test added for these breadcrumb items.

`http://localhost:8080/jenkins/manage/computer/(built-in)/configure`:
image

`http://localhost:8080/jenkins/manage/restart/`:
image

`http://localhost:8080/jenkins/manage/safeRestart/`:
image

`http://localhost:8080/jenkins/fingerprintCheck`:
image

`http://localhost:8080/jenkins/legend`:
image

`http://localhost:8080/jenkins/newView`:
image

`http://localhost:8080/jenkins/projectRelationship`:
image

(no idea where to find the legacyApiTokenAdministrativeMonitor one...)

### Proposed changelog entries

- Entry 1: Add missing breadcrumb items in resources/jenkins
- […]

### Proposed upgrade guidelines

N/A

### Submitter checklist

- [X] The Jira issue, if it exists, is well-described.
- [X] The changelog entries and upgrade guidelines are appropriate for the audience affected by the change (users or developers, depending on the change) and are in the imperative mood (see [examples](https://github.com/jenkins-infra/jenkins.io/blob/master/content/_data/changelogs/weekly.yml)).
  - Fill in the **Proposed upgrade guidelines** section only if there are breaking changes or changes that may require extra steps from users during upgrade.
- [X] There is automated testing or an explanation as to why this change has no tests.
- [ ] New public classes, fields, and methods are annotated with `@Restricted` or have `@since TODO` Javadocs, as appropriate.
- [ ] New deprecations are annotated with `@Deprecated(since = "TODO")` or `@Deprecated(forRemoval = true, since = "TODO")`, if applicable.
- [ ] New or substantially changed JavaScript is not defined inline and does not call `eval` to ease future introduction of Content Security Policy (CSP) directives (see [documentation](https://www.jenkins.io/doc/developer/security/csp/)).
- [ ] For dependency updates, there are links to external changelogs and, if possible, full differentials.
- [ ] For new APIs and extension points, there is a link to at least one consumer.

### Desired reviewers

@mention

### Maintainer checklist

Before the changes are marked as `ready-for-merge`:

- [ ] There are at least two (2) approvals for the pull request and no outstanding requests for change.
- [ ] Conversations in the pull request are over, or it is explicit that a reviewer is not blocking the change.
- [ ] Changelog entries in the pull request title and/or **Proposed changelog entries** are accurate, human-readable, and in the imperative mood.
- [ ] Proper changelog labels are set so that the changelog can be generated automatically.
- [ ] If the change needs additional upgrade steps from users, the `upgrade-guide-needed` label is set and there is a **Proposed upgrade guidelines** section in the pull request title (see [example](https://github.com/jenkinsci/jenkins/pull/4387)).
- [ ] If it would make sense to backport the change to LTS, a Jira issue must exist, be a _Bug_ or _Improvement_, and be labeled as `lts-candidate` to be considered (see [query](https://issues.jenkins.io/issues/?filter=12146)).

<a href="https://gitpod.io/#https://github.com/jenkinsci/jenkins/pull/7487"></a>

### CODE REVIEW

**Review by janfaracik — APPROVED** [review]

File: core/src/main/resources/jenkins/security/apitoken/LegacyApiTokenAdministrativeMonitor/manage.jelly
Comment (timja) [inline core/src/main/resources/jenkins/security/apitoken/LegacyApiTokenAdministrativeMonitor/manage.jelly:30]: should this be:

```suggestion
        <l:breadcrumb title="${%title}" />
```

File: core/src/main/resources/jenkins/model/Jenkins/configure.jelly
Comment (timja) [inline core/src/main/resources/jenkins/model/Jenkins/configure.jelly:32]: what does this add?
```suggestion
```

Comment (lemeurherve) [reply core/src/main/resources/jenkins/model/Jenkins/configure.jelly:32]: I didn't reproduced the body of my original PR, I should.

Here is the relevant bit:

> I've also added  as comment in views where adding a breadcrumb item wasn't necessary, let me know if it's preferable to omit them.

See also Wadeck comment about it: https://github.com/jenkinsci/jenkins/pull/7457#issuecomment-1330975869

Fine for me not adding it.

Comment (timja) [reply core/src/main/resources/jenkins/model/Jenkins/configure.jelly:32]: to me this seems like a development comment it also seems inaccurate as there is a breadcrumb on this page:

image

Comment (lemeurherve) [reply core/src/main/resources/jenkins/model/Jenkins/configure.jelly:32]: I had initially a longer comment, WDYT of:
```suggestion
  <!-- no need for additional breadcrumb here because we're on a page with an existing "Configure" breadcrumb -->
```

Comment (timja) [reply core/src/main/resources/jenkins/model/Jenkins/configure.jelly:32]: This still doesn't make sense to me because it's using the `f:breadcrumb-config-outline` which adds a breadcrumb to the page. I don't think a comment makes sense when you are using a tag that adds a breadcrumb, the docs on the tag itself explain more about what it does.

Comment (lemeurherve) [reply core/src/main/resources/jenkins/model/Jenkins/configure.jelly:32]: Sorry I didn't noticed that tag before, thanks, I agree with you then.

```suggestion
```

File: core/src/main/resources/jenkins/model/Jenkins/projectRelationship-help.jelly
Comment (timja) [inline core/src/main/resources/jenkins/model/Jenkins/projectRelationship-help.jelly:29]:
```suggestion
    <l:breadcrumb title="${%Title}" />
```

File: core/src/main/resources/jenkins/model/Jenkins/legend.jelly
Comment (timja) [inline core/src/main/resources/jenkins/model/Jenkins/legend.jelly:33]:
```suggestion
    <l:breadcrumb title="${%title}" />
```

File: core/src/main/resources/jenkins/model/Jenkins/configure.jelly
Comment (timja) [reply core/src/main/resources/jenkins/model/Jenkins/configure.jelly]: I've created a docs ticket: https://github.com/jenkinsci/design-library-plugin/issues/182

File: core/src/main/resources/jenkins/model/Jenkins/projectRelationship-help.jelly
Comment (lemeurherve) [reply core/src/main/resources/jenkins/model/Jenkins/projectRelationship-help.jelly:29]: LGTM

image

**Review by NotMyFault — APPROVED** [review]
Breadcrumbs for last views have been restored in https://github.com/jenkinsci/jenkins/commit/de352245d703fddbcde424c327c21e610748ad01, but I would still want to go ahead with this series of PRs, and add breadcrumbs to views, which don't have any yet.

**Review by timja — APPROVED** [review]
/label ready-for-merge

----

This PR is now ready for merge, after ~24 hours, we will merge it if there's no negative feedback.

Thanks!

**Pull request merged**
